ui/app/components/tabs/tab/tab.component.js

```diff
@@ -3,13 +3,13 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 
 const Tab = props => {
-  const { name, onClick, isActive, tabIndex } = props
+  const { name, onClick, isActive, tabIndex, className, activeClassName } = props
 
   return (
     <li
       className={classnames(
-        'tab',
-        isActive && 'tab--active',
+        className || 'tab',
```

Comment:
Let's move `'tab'` and `tab--active` to `defaultProps` on the component and then use `className` and `activeClassName` (below) directly.